qiita_db/reference.py

```diff
@@ -171,6 +171,10 @@ def tree_fp(self):
                         JOIN qiita.{0} r ON r.tree_filepath=f.filepath_id
                      WHERE r.reference_id=%s""".format(self._table)
             qdb.sql_connection.TRN.add(sql, [self._id])
-            rel_path = qdb.sql_connection.TRN.execute_fetchlast()
+            try:
```

Comment:
There is a test for this: Having the Silva new reference. That why I catch that this didn't work.